vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -1,28 +1,43 @@
 <template>
-    <div class="form-input">
-        <label-ripe class="header" v-bind:text="header" v-bind:for="id" v-if="header" />
-        <div class="content">
-            <slot />
-        </div>
+    <div class="form-input" v-bind:class="alignment">
         <label-ripe
-            class="footer"
-            v-bind:size="'small'"
-            v-bind:text="footer"
+            class="header"
+            v-bind:style="headerStyle"
+            v-bind:text="header"
             v-bind:for="id"
-            v-if="footer"
+            v-if="header"
         />
-        <div class="text-container">
-            <div class="text" v-if="error">
-                <span class="icon icon-error" />
-                <label-ripe v-bind:text="error" v-bind:color="'error'" />
-            </div>
-            <div class="text" v-if="warning">
-                <span class="icon icon-warning" />
-                <label-ripe v-bind:text="warning" />
-            </div>
-            <div class="text" v-if="success">
-                <span class="icon icon-success" />
-                <label-ripe v-bind:text="success" v-bind:color="'success'" />
+

```

Comment:
Did you `npm run pretty`? There shouldn't be blank lines.

Reply:
Yeah a did, but I doesn't remove the line :/